templates/website/src/app/(frontend)/[slug]/page.tsx

```diff
@@ -81,8 +81,8 @@ export default async function Page({ params: paramsPromise }: Args) {
   )
 }
 
-export async function generateMetadata({ params: paramsPromise }): Promise<Metadata> {
-  const { slug = 'home' } = await paramsPromise
+export async function generateMetadata({ params }: Args): Promise<Metadata> {
+  const { slug = 'home' } = await params
```

Comment:
Small nitpick here, but we should consistently rename `params` to `paramsPromise`. Since it's a paradigm shift, it's better they understand this is a promise you need to await, for those learning nextjs.